bitnami/kong/README.md

```diff
@@ -0,0 +1,326 @@
+| `securityContext.enabled`              | Enable securityContext on for Granafa deployment                                                       | `true`                                                  |
+| `securityContext.runAsUser`            | User for the security context                                                                          | `1001`                                                  |
+| `securityContext.fsGroup`              | Group to configure permissions for volumes                                                             | `1001`                                                  |
+| `securityContext.runAsNonRoot`         | Run containers as non-root users                                                                       | `true`                                                  |
+| `nodeSelector`                         | Node labels for pod assignment                                                                         | `{}`                                                    |
+| `tolerations`                          | Tolerations for pod assignment                                                                         | `[]`                                                    |
+| `affinity`                             | Affinity for pod assignment                                                                            | `{}`                                                    |
+| `podAnnotations`                       | Pod annotations                                                                                        | `{}`                                                    |
+| `sidecars`                             | Attach additional containers to the pod (evaluated as a template)                                                                                         | `nil`                                                                                                   |             |
+| `initContainers`                       | Add additional init containers to the pod (evaluated as a template)                                                                                       | `nil`                                                                                                   |             |
+| `podDisruptionBudget.enabled`                       | Deploy a PodDisruptionBudget object for the Kong pod                                                                                       | `false`                                                                                                   |             |
+| `podDisruptionBudget.maxUnavailable`                       | Maximum unavailable Kong replicas (expressed in percentage)                                                                                       | `50%`                                                                                                   |             |
```

Comment:
Suggestion. Please note you'll need to adapt the rest of manifests to these changes.
```suggestion
| `pdb.create` | Enable/disable a Pod Disruption Budget creation for Kong | `false` |
| `pdb.minAvailable` | Minimum number/percentage of pods that should remain scheduled | `nil` |
| `pdb.maxUnavailable` | Maximum number/percentage of pods that may be made unavailable | `50%` |
```

Reply:
Done